assemblyline/templates/ui.yaml

```diff
@@ -40,6 +40,7 @@ spec:
         app: assemblyline
         section: core
         component: ui
+        azure.workload.identity/use: true
```

Comment:
Wonder if this should be a value in the `coreLabels`, would there be a problem with this label being applied to all core components? 

Understandably only the container doing the authentication with the IdAM would need this label but I would assume it could be harmless for other containers to have this label?